api/src/params.ts

```diff
@@ -0,0 +1,9 @@
+export const params = {
+  MASTER_ADMIN: "dappnode_admin",
+  DAPPNODE_INTERNAL_IP: "http://my.dappnode/global-envs/INTERNAL_IP",
+  SERVER_PORT: 51820,
```

Comment:
The port of what server? Why such a high port number?

Reply:
seems to be something native of wireguard 

https://github.com/linuxserver/docker-wireguard/search?q=51820